src/main/scala/ru/gaptrakhmanov/telegram/helper/commands/wordgame/ValidateWordService.scala

```diff
@@ -0,0 +1,23 @@
+package ru.gaptrakhmanov.telegram.helper.commands.wordgame
+
+import cats.effect.Sync
+import cats.implicits._
+import io.circe.generic.auto._
+import io.circe.parser
+import ru.gaptrakhmanov.telegram.helper.util.UrlUtils
+
+object ValidateWordService {
+
+  private case class Word(word: String)
+
+  private val wordDescriptionUrl = "https://api.dictionaryapi.dev/api/v2/entries/en/%s"
+
+  def isRealEnglishWord[F[_] : Sync](word: String): F[Boolean] = for {
+    wordDescriptionJson <- UrlUtils.getDataFromUrl[F](String.format(wordDescriptionUrl, word))
+      .handleError(_ => "Error while getting word description!")
```

Comment:
One suggestion here could be:

- Log the error that sometimes helps when we want to debug why we are getting "Error while getting word description!".

Reply:
Sure, I think that we need extra task to add logging to whole project.